tests/ui/consts/ice-const-size-relate-126359.stderr

```diff
@@ -0,0 +1,18 @@
+error[E0308]: mismatched types
+  --> $DIR/ice-const-size-relate-126359.rs:12:39
+   |
+LL |     let _ = OppOrder::<3, u32> { arr: [0, 0, 0] };
+   |                                       ^^^^^^^^^ expected `3`, found `3`
+   |
+   = note: expected array `[u32; 3]`
+              found array `[u32; 3]`
```

Comment:
wooondering if its worth having a `debug_assert` against this kind of stuff ('note: expected X, found Y' where x.display() == y.display()) because that just seems silly from a diagnostic point of view :sweat_smile: